src/sentry/ingest/userreport.py

```diff
@@ -220,6 +226,33 @@ def should_filter_user_report(
                     "feedback_message": comments[:100],
                 },
             )
-        return True, "Too Large"
+        return True, "too_large.message", "Message Too Large"
+
+    max_name_length = UserReport._meta.get_field("name").max_length
+    if max_name_length and len(name) > max_name_length:
+        return True, "too_large.name", "Name Too Large"
+
+    max_email_length = UserReport._meta.get_field("email").max_length
+    if max_email_length and len(email) > max_email_length:
+        return True, "too_large.email", "Email Too Large"
+
+    try:
+        if email:
+            validate_email(email)
+    except ValidationError:
+        return True, "invalid_email", "Invalid Email"
+
+    try:
+        # Validates UUID and strips dashes.
+        report["event_id"] = uuid.UUID(report["event_id"].lower()).hex
+    except ValueError:
+        return True, "invalid_event_id", "Invalid Event ID"
+
+    if options.get("feedback.filter_garbage_messages"):  # Message-based filter kill-switch.
+        if not comments:
+            return True, "empty", "Empty Feedback Messsage"
+
+        if comments == UNREAL_FEEDBACK_UNATTENDED_MESSAGE:
+            return True, "unreal.unattended", "Sent in Unreal Unattended Mode"
```

Comment:
changing the order that this runs could change the metrics & output. What if someone has a large name or email, and it's also an empty message?